core/modules/flowgen.cc

```diff
@@ -381,8 +411,8 @@ CommandResponse FlowGen::Init(const bess::pb::FlowGenArg &arg) {
     return CommandFailure(ENOMEM, "task creation failed");
   }
 
-  templ_ = new char[MAX_TEMPLATE_SIZE];
-  if (templ_ == nullptr) {
+  tmpl_ = new char[MAX_TEMPLATE_SIZE];
```

Comment:
Hm... although it's an existing issue and not part of this PR, I am not sure why `tmpl_` is dynamically allocated here. Could you please make it `unsigned char tmpl_[MAX_TEMPLATE_SIZE];` member of the module class?